src/tracking.js

```diff
@@ -0,0 +1,8 @@
+var idSite = 8
+var piwikTrackingApiUrl = 'https://piwik.cozycloud.cc/piwik.php'
```

Comment:
You can't replace data with webpack compilation variables (as defined in the `Define` plugin) when you _copy_ the file. You must `require` it to take care of the defined compilation vars.

The point is that we can here not copying the `tracking.js` file but require it in `main.jsx|router` to enable the tracking. As the tracking file rely on the global `_paq` variable, we can use the [`external` config](https://webpack.github.io/docs/library-and-externals.html) key to declares

```js
externals: {
  "_paq": "_paq"
}
```

and do a `require('_paq') in `tracking.js` to use the `_paq` var injected by the `piwiki.js` external script.